### ISSUE TITLE
Add full table of properties and what CSSStyleValues types they normalize to

### ISSUE THREAD

**Issue by nainar** [w3c/css-houdini-drafts#488]

**Comment by nainar** [comment]
Yes, a map of properties to Typed OM objects they normalize to should be added into the spec.

**Comment by dbaron** [comment]
It seems like this needs to be done in a way similar to how we've extended property-specific data in the past (such as for interpolation behavior):  the Typed OM spec should specify things for existing properties, and specify the mechanism that new properties add themselves.

**Comment by nainar** [comment]
There is now an initial table of about ~60 or so popular properties with the StyleValues their specified and computed values normalize to.

**Comment by nainar** [comment]
dbaron@ is it ok to close this since we have the more popular properties here? Or do you want to close this when we have a list of all properties?

**Comment by tabatkins** [comment]
I'd rather leave it open until the table is complete.
